app/views/shared/_todo_list.html.erb

```diff
@@ -0,0 +1,9 @@
+<ul>
```

Comment:
These two loops look really similar. Perhaps you could refactor this? I think I remember Kevin saying this morning you could concatenate the arrays, something like `@incomplete_todos + @completed_todos.each do |todo|`.

Reply:
We considered making that change but decided we preferred the semantic separation of the two lists. I wonder if there is a performance difference.